### PR TITLE
Round icons on the list

### LINKED ISSUE #34: blockies should be always masked to a circle

### PR DESCRIPTION
**Pull request by tomusdrw**
Closes #34 

Also adds an intro text to create a new account if you have none.
screen
screen

### CODE REVIEW

File: src/components/AccountListRow.js [hunk 1]
```diff
@@ -15,9 +16,9 @@ export default class AccountListRow extends Component {
     return (
       <TouchableHighlight style={styles.row} onPress={this.props.onPress} underlayColor='#0004'>
         <View style={{flexDirection: 'column'}}>
-          <View style={{flexDirection: 'row'}}>
-            <AccountIcon style={styles.square} seed={this.props.lowerText} />
-            <View style={{flexDirection: 'column'}}>
+          <View style={styles.innerRow}>
+            <AccountIcon style={[AppStyles.icon, styles.icon]} seed={this.props.lowerText} />
```
Comment (debris) [inline src/components/AccountListRow.js]: you can pass styles as an array? cool! I didn't know about that ;)

Comment (tomusdrw) [inline src/components/AccountListRow.js]: Yup, saw that in examples in docs :)

File: src/components/AccountList.js [hunk 2]
```diff
@@ -28,6 +29,26 @@ export default class AccountList extends Component {
   }
 
   render () {
+    if (!this.props.accounts.length) {
+      return (
+        <View style={AppStyles.view}>
+          <View style={styles.introContainer}>
+            <Text style={styles.introText}>
+              To sign transactions you need at least one account.
```
Comment (debris) [inline src/components/AccountList.js]: wanted to leave a comment about redundant `will`, but it's gone ;)

**Comment by debris** [comment]
lgtm! :)

**Pull request merged**